README.md

```diff
@@ -203,6 +203,7 @@ Mysql2::Client.new(
   :reconnect = true/false,
   :local_infile = true/false,
   :secure_auth = true/false,
+  :ssl_mode = "DISABLED" | "PREFERRED" | "REQUIRED" | "VERIFY_CA" | "VERIFY_IDENTITY",
```

Comment:
I was thinking that these could be symbols, `:disabled` (or `false`), `:preferred`, `:required`, `:verify_ca`, `:verify_identity`. That said, there's already `:ssl_verify` which means `:verify_ca` - there's a bunch of overlap that should be resolved as much as possible.